actor/src/main/scala/org/apache/pekko/dispatch/ForkJoinExecutorConfigurator.scala

```diff
@@ -86,15 +87,28 @@ class ForkJoinExecutorConfigurator(config: Config, prerequisites: DispatcherPrer
   }
 
   class ForkJoinExecutorServiceFactory(
+      val id: String,
```

Comment:
useless field?

Reply:
used in `createExecutorServiceFactory`, some lines below.